src/monitor/monitor.c

```diff
@@ -2180,7 +2180,7 @@ static void monitor_service_shutdown(struct mt_svc *svc)
 
     /* We must decrease the number of services when shutting down
      * a {socket,dbus}-activated service. */
-    ctx->num_services--;
+    if (ctx->num_services) ctx->num_services--;
```

Comment:
Hi,

from the description in #5598 I wonder if you mean `if (ctx) ....` or, to be more explicit, `if (ctx != NULL) ...`?

bye,
Sumit

Reply:
Of course. Thank you.